content/applications/sales/sales/products_prices/loyalty_discount.rst

```diff
@@ -0,0 +1,189 @@
+=============================
+Discount and loyalty programs
+=============================
+
+The Odoo *Sales*, *eCommerce*, and *Point of Sale* applications allow users to create discount and
+loyalty programs that customers can use for online and in-store shopping. These programs offer more
+varied, public, and time-sensitive pricing options than :doc:`pricelists
+</applications/sales/sales/products_prices/prices/pricing>`.
+
+Configure the settings
+======================
+
+To begin using discount and loyalty programs, navigate to :menuselection:`Sales --> Configuration
+--> Settings`. Under the :guilabel:`Pricing` heading, activate the :guilabel:`Discounts, Loyalty &
+Gift Card` setting by checking the box next to the feature. Finally, click :guilabel:`Save` to save
+the changes.
+
+Configure discount and loyalty programs
+=======================================
+
+To create discount and loyalty programs, go to :menuselection:`Sales --> Products --> Discount &
+Loyalty`.
+
+If no discount or loyalty programs have been created yet, Odoo provides a choice of templates to
+help create the first program. Choose one of the template cards, or click :guilabel:`New` to create
+a new program from scratch.
+
+Or, if there are already existing programs, select an existing program to edit it.
+
+.. image:: loyalty_discount/price-discount-loyalty.png
+   :align: center
+   :alt: Discount and loyalty program template cards.
+
+.. note::
+   Templates only appear when no programs have been created, and they disappear once the first
+   program is created.
+
+Creating or editing a program opens the program form.
+
+The program form contains the following options:
+
+- :guilabel:`Program Name`: Enter the name of the program. The program name is *not* visible to the
```

Comment:
This is a matter of style, and depends on which guide you follow (i.e. APA vs. Chicago, etc.), however we've been leaving the first word after a colon *uncapitalized* and do not add periods at the end of list items.

However, what matters most is consistency, so the way this list is formatted works, as well 😉  
No need to change if you don't want to, but thought it was worth mentioning.
(The form walk-through is nice btw!)